### ISSUE TITLE
Scaffold Paginator breaks CSS media queries

### ISSUE THREAD

**Issue by rwetzlmayr** [SocialEngine/phpv4-issues#214]
Applies to SE 4.8.9.

Steps to reproduce:
1. Switch your site into development mode. Scaffold may serve cached style sheets otherwise.
2. Upload this large CSS file to your site: https://gist.github.com/rwetzlmayr/4de46a2090a28b7ade76. For illustration purposes I will use the `temporary` folder and the file name `fubar.css` in this issue report. You may use any other suitable folder and filename.  Note that this file includes media queries and contains more than 4000 selectors. 
3. Visit http://example.com/application/css.php?request=temporary%2Ffubar.css
4. The first line contains an `@import` statement referring to `/application/css.php?request=temporary%2Ffubar.css&pageStart=99860&pageEnd=102490`. Open that style sheet from http://example.com/application/css.php?request=temporary%2Ffubar.css&pageStart=99860&pageEnd=102490.
5. Notice that the imported CSS file does no longer frame the style rules with a media query and we now have a lot of global CSS rules which probably weren't intended for current output device. Also, there's a dangling right brace at the very end of that imported style sheet.

Root cause:

The `Paginator` module naively looks for right braces to find a segmentation breakpoint in `Paginator::_getPreviousSelectorEnd()`. This is not a valid assumptions as soon as media queries are allowed in style sheets.

**Comment by smazurov** [comment]
Yes, this is a limitation of our css.php module. As we're shipping a responsive theme as part of 4.9.0, this will be addressed in the near future. See #150 for more details on responsive theme.

**Comment by alimoosavi15** [comment]
a quick fix without modifying any files would be creating this file under : application/libraries/Scaffold/modules/Inline/Inline.php

```
<?php

class Inline
{
  
	public static function initialize()
	{
		
		// Ignore paginator to Fix SocialEngine bug in paginator cssaffold module
		@$_GET['pageStart'] = 1;
		
	}
	
?>

**Comment by Elshara** [comment]
I support this. It's best not to put your site in development mode it could leave you unable to access your site at all.

**Comment by Suxxen** [comment]
I tried the above quick fix on SE, but it seems to have completely broken the CSS now.  Am I missing something?

**Comment by DonnaScriptTechs** [comment]
This is fixed for 4.9 when we release it.

**Comment by Elshara** [comment]
Awesome.
